cst438-homework1/src/main/java/cst438/MovieRating.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Movie Rating class for creating the rating objects
+ */
+
+package cst438;
+
+import java.util.Date;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Entity
+public class MovieRating {
+
+  @Id
+  @GeneratedValue
+  private long id;
+
+  @NotNull
+  @Size(min = 3, max = 25)
+  private String title;
+
+  @NotNull
```

Comment:
I would recommend adding a minimum and maximum constraint here. 
@min(1)
@max(5)
See the link below for further documentation on constraints.
https://docs.oracle.com/javaee/7/tutorial/bean-validation001.htm

Reply:
I decided to put constraints on the front end instead, but its probably smart to cover all bases and do both!